src/components/Navbar.tsx

```diff
@@ -76,11 +76,11 @@ const Navbar = () => {
 						onClick={handleLogoReset}
 						className={classes.logo}
 						elevation={0}
-						sx={{ my: 2, color: "black", display: "block", textTransform: "unset !important", fontFamily: "Avenir Next", minHeight: "0", minWidth: "0", padding: "0" }} 
+						sx={{ my: 2, color: "black", display: "block", textTransform: "unset !important", fontFamily: "Avenir Next", minHeight: "0", minWidth: "0", padding: "0" }}
```

Comment:
**thought:** I am usually very careful with using `!important` and overwriting MUI styles this way, it may become very dirty in the long run and hard to maintain. I am usually trying to backtrack the style in the dev console and see what other proprieties are blocking the property I try to change. It's usually a parent or a global prop blocking it from usage, so I try to remove it from the parents and assign it only to the right children. I find exploration of design systems very fun